pkg/rebuild/rebuild/rebuildremote.go

```diff
@@ -110,39 +127,71 @@ var assetUploadTpl = template.Must(
 	))
 
 func makeBuild(t Target, dockerfile string, opts RemoteOptions) (*cloudbuild.Build, error) {
+	var finalizeScript bytes.Buffer
+	{
+		err := rebuildFinalizeTpl.Execute(&finalizeScript, map[string]any{
+			"Artifact": transfer{
+				From: path.Join("/out", t.Artifact),
+				To:   path.Join("/workspace", t.Artifact),
+			},
+			"EnableEBPF": opts.EnableEBPF,
+		})
+		if err != nil {
+			return nil, errors.Wrap(err, "expanding asset upload template")
+		}
+	}
 	var assetUploadScript bytes.Buffer
-	err := assetUploadTpl.Execute(&assetUploadScript, map[string]any{
-		"UtilPrebuildBucket": opts.UtilPrebuildBucket,
-		"Uploads": []upload{
+	{
+		var uploads = []transfer{
 			{From: "/workspace/image.tgz", To: opts.RemoteMetadataStore.URL(Asset{Target: t, Type: ContainerImageAsset}).String()},
 			{From: path.Join("/workspace", t.Artifact), To: opts.RemoteMetadataStore.URL(Asset{Target: t, Type: RebuildAsset}).String()},
-		},
-	})
-	if err != nil {
-		return nil, errors.Wrap(err, "expanding asset upload template")
+		}
+		if opts.EnableEBPF {
+			uploads = append(uploads, transfer{From: "/workspace/tetragon.jsonl", To: opts.RemoteMetadataStore.URL(Asset{Target: t, Type: TetragonLog}).String()})
+		}
+		err := assetUploadTpl.Execute(&assetUploadScript, map[string]any{
+			"UtilPrebuildBucket": opts.UtilPrebuildBucket,
+			"Uploads":            uploads,
+		})
+		if err != nil {
+			return nil, errors.Wrap(err, "expanding asset upload template")
+		}
 	}
+	steps := []*cloudbuild.BuildStep{}
+	if opts.EnableEBPF {
+		steps = append(steps, &cloudbuild.BuildStep{
+			Name:         "gcr.io/cloud-builders/docker",
+			Args:         []string{"run", "--name=tetragon", "--pid=host", "--cgroupns=host", "--privileged", "-v=/sys/kernel/btf/vmlinux:/var/lib/tetragon/btf", "quay.io/cilium/tetragon:v1.1.2", "/usr/bin/tetragon", "--export-filename=/var/log/tetragon/tetragon.jsonl"},
```

Comment:
No good reason, do you have a prefered place to put it?

Should I save it directly to /workspace/tetragon.jsonl?

Reply:
I'm honestly not sure...

While this definitely fits the purpose and shape of data in /var/log, I'm also hesitant to put it there if we want a very specific behavior from the log file. I think both could work so I'll defer to you.

Reply:
Moved to  /workspace/tetragon.jsonl which I `touch` then mount directly into tetragon (to avoid exposing the entire /workspace directory to tetragon).